docs/000-contributing/writing-style-guide.md

```diff
@@ -4,6 +4,16 @@
 
 This will be fleshed out in future iterations. For now, keep in mind that content appearing in the handbook should be welcoming, personable, and free from jargon.
 
+## How to capitalize titles and headings
+
+We use sentence case for our titles and headings. When using sentence case, we capitalize the first word, all proper nouns, and a word following a colon. We do not capitalize the next word if a title or heading begins with a number.
+
+Examples
+
+- Site building using Drupal
+- Engineer's role in client relationships
+- 30 days checkin
```

Comment:
I think it is either "check in" or "check-in" but not "checkin"

Reply:
Thanks for catching that.